app/assets/index.html

```diff
@@ -20,5 +20,6 @@
 </div>
 
 <script src="scripts/main.js"></script>
+<script>require('app/scripts/main');</script>
```

Comment:
This line should not be necessary; `main` is automatically required (see line 19 of `brunch-config.js`).